controllers/search_controller.go

```diff
@@ -76,7 +76,7 @@ var cleanOnce sync.Once
 //+kubebuilder:rbac:groups=addon.open-cluster-management.io,resources=managedclusteraddons/status;clustermanagementaddons/status,verbs=get;update;patch
 //+kubebuilder:rbac:groups=authentication.open-cluster-management.io,resources=managedserviceaccounts,verbs=create;get;delete
 //+kubebuilder:rbac:groups=cluster.open-cluster-management.io,resources=managedclusters,verbs=get;list
-//+kubebuilder:rbac:groups=operator.open-cluster-management.io,resources=multiclusterglobalhubs,verbs=get;list
+//+kubebuilder:rbac:groups=operator.open-cluster-management.io,resources=multiclusterglobalhubs;multiclusterhubs,verbs=get;list
```

Comment:
Need to run `make bundle` to propagate to `bundle/manifests/search-v2-operator.clusterserviceversion.yaml`

**Note it generates about 5 changes, please only commit the 1 change that's relevant for this change.**